pipeline/trusted-resources/examples/1-test-taskrun.yaml

```diff
@@ -0,0 +1,13 @@
+kind: TaskRun
+apiVersion: tekton.dev/v1beta1
+metadata:
+  annotations:
+      tekton.dev/signature: MEYCIQDhKiJpPylEFo5RmBEZV96luADJdhSQcE7EZuOgL7hk8wIhAICKk6o5ldqo2sN0R6GhE7GpuEkblbjeJilNms78E+ap
```

Comment:
How are these generated? (e.g. if I modify this or want to create another, what do I need to do?)

Reply:
This is just an example showing how to use taskrun with this webhook deployed (also for demo and manually tetsting). 
I'm not sure if we need to include the signing code just for generating the signatures? I can add them into a folder like `etc/...`

Reply:
So right now it is user's responsibility to create such yaml files to my understanding

Reply:
Or I just comment out these signatures and add some comments?

Reply:
It's fine to have the signatures here - the missing piece is how these were generated.

At minimum, we should have a readme to document how to generate these. If you want to include a tool to make this easier that works to!